server/settings/publish_plugins.py

```diff
@@ -157,6 +159,77 @@ def validate_unique_outputs(cls, value):
         return value
 
 
+class ResolutionOptionsModel(BaseSettingsModel):
+    _layout = "compact"
+    width: int = SettingsField(
+        1920,
+        ge=0,
+        le=100000,
+        title="Width",
+        description=(
+            "Width resolution number value"),
+        placeholder="Width"
+    )
+    height: int = SettingsField(
+        1080,
+        title="Height",
+        ge=0,
+        le=100000,
+        description=(
+            "Height resolution number value"),
+        placeholder="Height"
+    )
+    pixel_aspect: float = SettingsField(
+        1.0,
+        title="Pixel aspect",
+        ge=0.0,
+        le=100000.0,
+        description=(
+            "Pixel Aspect resolution decimal number value"),
+        placeholder="Pixel aspect"
+    )
+
+
+def ensure_unique_resolution_option(
+        objects: Iterable[Any], field_name: str | None = None) -> None:  # noqa: C901
```

Comment:
I think it will be always `list`.
```suggestion
        objects: list[Any], field_name: str | None = None) -> None:  # noqa: C901
```

Reply:
Why is expecting `field_name` if is not used?